src/components/table-ajax/table-ajax.js

```diff
@@ -327,6 +335,10 @@ class TableAjax extends Table {
       const data = response.body;
       this.props.onChange(data);
       this.setState({ totalRecords: String(data.records) });
+    } else if (this.props.onAjaxError) {
+      this.props.onAjaxError(err, response);
+    } else {
+      console.warn(`${err.status} - ${response}`); // eslint-disable-line no-console
```

Comment:
using the Logger could be a good idea as it won't output anything in production right?